src/main/java/Duke.java

```diff
@@ -1,10 +1,84 @@
+import duke.exception.DukeException;
+import duke.iomethods.IOMethods;
+import duke.task.TaskManager;
+import duke.fileHandler.FileHandler;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
 public class Duke {
-    public static void main(String[] args) {
+
+    //fields
+    private static TaskManager taskManager = new TaskManager();
+
+    //methods
+    public static void hello() {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        IOMethods.printWithDivider("Hello! I'm Duke\nWhat can I do for you?");
+    }
+
+
+    //main
+    public static void run(String line) throws DukeException {
+        ArrayList<String> words = IOMethods.splitToTwo(line, " ");
+        String command = words.get(0);
+        String description = words.size()>= 2 ? words.get(1) : null;
```

Comment:
`2` is a [magic number ](https://nus-cs2113-ay2122s1.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-magic-numbers)that you could avoid. Consider using named constants instead.